### ISSUE TITLE
Would you consider changing the license to something that is easier to integrate with other packages?

### ISSUE THREAD

**Issue by diekhans** [althonos/pronto#14]
Pronto is awesome; py2+3, easy to use. However difficult for us to adopt due to the license.

GPL causes a lot of grief because of it forcing GPL on all software using it.  This isn't a matter of wanting to sell software, it's about making it as easily available as possible.   We have gone the Apache license route for just this reason.

Would you consider LGPL 3 (or anything else less invasive)?

**Comment by althonos** [comment]
Hi there, first of all I'm glad you like the library.

To be honest, when I started this library by outsourcing some of the code I was developing for a research project. At that time I didn't have a lot of experience in licenses, and since we went with GPL for that project, I went with GPL for `pronto` too.

So I will probably end up relicensing this library to MIT, because I went with it a while back, and I'm fine with my code being used as long as there's my name somewhere in it 😄
